sdk/computefleet/tests.mgmt.yml

```diff
@@ -0,0 +1,17 @@
+trigger: none
+
+pr: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: computefleet
+    Artifacts:
+      - name: azure-resourcemanager-computefleet
+        groupId: com.azure.resourcemanager
+        safeName: azureresourcemanagercomputefleet
+    Clouds: 'Public'
+    UseFederatedAuth: true
```

Comment:
@weidongxu-microsoft where do we generate these files? I would like to update them to remove "Clouds" and "UseFederatedAuth" parameters because they are no necessary.

Reply:
I should be in some script under eng.

@XiaofeiCao Can you update the script and do a clean up on existing ci files?